giraffe/src/components/Axes.tsx

```diff
@@ -100,9 +100,18 @@ export const drawAxes = ({
   context.textAlign = 'end'
   context.textBaseline = 'middle'
   const yDomainWidth = yDomain[1] - yDomain[0]
+  if (yColumnLabels) {
+    context.globalAlpha = 1
+    context.fillStyle = tickFontColor
+    for (let i = 0; i < yColumnLabels.length; i++) {
+      const y = yScale(i) + margins.top - height / (yColumnLabels.length * 2)
```

Comment:
might want to put a comment here saying it's possible to divide by zero here since `yColumnLabels.length` can never be `0` on this line. I noticed a potential divide by zero and had to work out that this is a safe division because of the loop above. A comment encodes that mental work into something a little less work for the readier :)